recipes/aws-sdk-cpp/all/test_package/aws-sdk-cpp-plugin/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.8)
 
 add_project(aws-sdk-cpp-plugin "C++ AWS SDK plugin" AWS::aws-sdk-cpp-s3)
```

Comment:
When building the branch locally, the `add_project` call fails within the test:
```
CMake Error at aws-sdk-cpp-plugin/CMakeLists.txt:3 (add_project):
  Unknown CMake command "add_project".
```

Reply:
It has been fixed by the commit https://github.com/conan-io/conan-center-index/pull/17125/commits/48548d8f31cfe32169766dfe713fd6a4bf1b79e5

Reply:
The error is now gone. Probably fixed with the improved cmake export.